cpp/src/Slice/Parser.h

```diff
@@ -1005,9 +1006,15 @@ class Enum : public virtual Container, public virtual Constructed
 public:
 
     virtual void destroy();
+
+    // The underlying numeric type. The default is varint. The only permissible explicit types are fixed-length
+    // numeric types such as byte, short, ushort.
```

Comment:
If we are indeed not going to support `long` and `ulong`, I think the comment should be updated to explicitly say they're excluded. Since right now they are fixed-length numeric types, and yet invalid for enumerators.

Or instead of updating the comment, we _could_ just add long support!

Reply:
For a future PR.